src/core/asyncProcess.js

```diff
@@ -13,14 +13,14 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2016-2019 (original work) Open Assessment Technologies SA ;
+ * Copyright (c) 2016-2024 (original work) Open Assessment Technologies SA ;
  */
 /**
  * @author Jean-Sébastien Conan <jean-sebastien.conan@vesperiagroup.com>
  */
 import _ from 'lodash';
-import Promise from 'core/promise';
-import eventifier from 'core/eventifier';
+import Promise from './promise';
+import eventifier from './eventifier';
```

Comment:
**suggestion:** to properly follow the requirements, we also need to add the file extension

```suggestion
import Promise from './promise.js';
import eventifier from './eventifier.js';
```